.eslintrc.js

```diff
@@ -20,7 +20,11 @@ module.exports = {
     "no-plusplus": [2, { allowForLoopAfterthoughts: true }],
     "no-empty-function": "off",
     "no-useless-constructor": "off",
-    "prettier/prettier": "error"
+    "prettier/prettier": "error",
+    "no-shadow": "off",
+    "no-unused-vars": "off",
```

Comment:
Is there any reason for ignoring these? `no-shadow` and `no-unused-vars` seem like pretty good rules?